docs/docker-image-details.md

```diff
@@ -118,6 +118,21 @@ You can use this kind of image to test the catalog of Apache Hive.
 
 Changelog
 
+- gravitino-ci-hive:0.1.13
+  - Enable Ranger plugin in the Hive and HDFS
+  - Hive version 3.1.3
+  - HDFS version 3.1.0
```

Comment:
Have you upgraded the versions of Hive and Hadoop from 2.x to 3.x?

All implementations and tests are based on version 2.x. I am concerned that upgrading to version 3.x may introduce potential compatibility issues that have not been tested.